lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -98,10 +99,12 @@ function runLighthouse(url, configPath, isDebug) {
     console.error(`STDERR: ${runResults.stderr}`);
   }
 
-  const lhr = fs.readFileSync(outputPath, 'utf8');
+  const lhr = runResults.status === PAGE_HUNG_EXIT_CODE ?
```

Comment:
since we don't really do anything else with it below (e.g. no path to debug output), could this be just an early return of the PAGE_HUNG LHR?

Reply:
yeah good point, done!